Description: document that socket activation voids bind_to_address setting
Author: Florian Schlichting <fsfs@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=785418
Forwarded: http://mailman.blarg.de/pipermail/mpd-devel/2015-June/000363.html

--- a/doc/mpdconf.example
+++ b/doc/mpdconf.example
@@ -75,7 +75,8 @@
 #
 # This setting sets the address for the daemon to listen on. Careful attention
 # should be paid if this is assigned to anything other then the default, any.
-# This setting can deny access to control of the daemon.
+# This setting can deny access to control of the daemon. Not effective if
+# systemd socket activiation is in use.
 #
 # For network
 #bind_to_address		"any"
